packages/gatsby-theme-newrelic/src/components/MarkdownContainer.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+
+const MarkdownContainer = ({
+  children,
+  className,
+  dangerouslySetInnerHTML,
+}) => {
+  return (
+    <div
+      dangerouslySetInnerHTML={dangerouslySetInnerHTML}
+      className={className}
```

Comment:
Since the keys equal the tag attributes, we could consider writing it like this:

```jsx
const MarkdownContainer = ({ children, ...props }) => (
  <div
    ...props
    css={`...`}
  >
    {children}
  </div>
);
```

That said, what you have hear is more descriptive. Just a thought.

Reply:
Thats true. Really the only difference between them is one style allows _all_ props (`onClick`, etc) vs this just allowing these 2 components.

I don't feel strongly either way. I'd say lets keep this for now and when the need arises to add support for more props, we can switch over to the spread props. 

How do you feel about that?